src/OrchardCore.Modules/OrchardCore.Features/Controllers/AdminController.cs

```diff
@@ -19,6 +19,7 @@
 
 namespace OrchardCore.Features.Controllers
 {
+    [Admin]
     public class AdminController : Controller
```

Comment:
Looks good @MatthijsKrempel but for the extra `Admin` attributes.

We have a well defined (if misunderstood) convention that it's an Admin controller if called `AdminController` or if has `[Admin]`.

I don't think we need to muddy the waters anymore by then putting the attribute on controllers which conform to the convention.

Also you only added it to some admin controllers, so more confusing ;)

Maybe I try and write some docs for it

Reply:
I can remove those if you want, I think it is more explicit to use the attribute rather then rely on a convention. But up to you guys.